internal/packages/packages.go

```diff
@@ -96,6 +97,11 @@ type Conditions struct {
 	Elastic ElasticConditions `config:"elastic" json:"elastic" yaml:"elastic"`
 }
 
+// Discovery define indications for the data this package can be useful with.
+type Discovery struct {
+	Fields []string `config:"fields" json:"fields" yaml:"fields"`
```

Comment:
According to the package-spec, this is an array of objects.
Example from package-spec:
```yaml
discovery:
  fields:
    - name: process.pid
```

Reply:
Fixed, thanks!